.github/workflows/downstream_tests.yml

```diff
@@ -228,14 +228,16 @@ jobs:
         run: |
           cd tea-tasting
           pdm remove narwhals
-          pdm add ./..[tests]
+          pdm add ./..
       - name: show-deps
         run: |
           cd tea-tasting
           pdm list
       - name: run-pytest
         run: |
           cd tea-tasting
+          # empty pytest.ini to avoid pytest using narwhals configs
+          touch pytest.ini
```

Comment:
lol this was fun to debug. 😅
`tea-tasting` doesn't have pytest configs in the `pyproject.toml` so `pytest` was using the `narwhals` config (and therefore requiring narwhals testing dependencies)

Can anyone think of a cleaner way to do this?